blackbox/docs/configuration.txt

```diff
@@ -493,6 +493,18 @@ Repositories are used to :ref:`backup <snapshot-restore>` a CrateDB cluster.
 
 See also the :ref:`path.repo <conf-path-repo>` Setting.
 
+Queries
+-------
+
+.. _conf-indices-query-bool.max_clause_count:
+
+**indices.query.bool.max_clause_count**
```

Comment:
for the user it is not really useful if this is not a runtime setting and he has to set it on each node, can we change this?